extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1261,17 +1260,21 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
     this.signals()
       .whenSignal(CommonSignals.LOAD_END)
-      .then(() => this.maybeApplyFirstAnimationFrame())
-      .then(() => {
-        this.animationManager_.finishAll();
-      });
+      .then(() => this.animationManager_.applyLastFrame());
+  }
+
+  /**
+   * @return {!Promise}
+   */
+  maybeApplyFirstAnimationFrameOrFinish() {
+    return Promise.resolve(this.animationManager_?.applyFirstFrameOrFinish());
   }
 
   /**
    * @return {!Promise}
    */
-  maybeApplyFirstAnimationFrame() {
-    return Promise.resolve(this.animationManager_?.applyFirstFrame());
+  maybeApplyLastAnimationFrame() {
```

Comment:
Is this being called anywhere?

Reply:
Good catch, I use `maybeApplyFirstAnimationFrameOrFinish` instead. Will remove it